examples/fl-tabular/centralized.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelEncoder, StandardScaler
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader, TensorDataset
+
+url = "https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data"
+column_names = [
+    "age",
+    "workclass",
+    "fnlwgt",
+    "education",
+    "education_num",
+    "marital_status",
+    "occupation",
+    "relationship",
+    "race",
+    "sex",
+    "capital_gain",
+    "capital_loss",
+    "hours_per_week",
+    "native_country",
+    "income",
+]
+
+data = pd.read_csv(
+    url, header=None, names=column_names, na_values=" ?", sep=",\s", engine="python"
+)
+
+data.dropna(inplace=True)
+
+label_encoders = {}
+for column in data.select_dtypes(include=["object"]).columns:
+    label_encoders[column] = LabelEncoder()
```

Comment:
`LabelEncoder` is not meant to be used on input data (X) but instead on target (y).
`OrdinalEncoder` should be used instead.

It also brings problems of different preprocessing of the dataset in centralized and federated settings. Why would be the reason for that? Also, no info about that in the README.